frontend/server/server.js

```diff
@@ -0,0 +1,168 @@
+// Uncomment following to enable zipkin tracing, tailor to fit your network configuration:
+// var appzip = require('appmetrics-zipkin')({
+//     host: 'localhost',
+//     port: 9411,
+//     serviceName:'frontend'
+// });
+
+require('appmetrics-dash').attach();
+require('appmetrics-prometheus').attach();
+const appName = require('./../package').name;
+const http = require('http');
+const express = require('express');
+const log4js = require('log4js');
+const localConfig = require('./config/local.json');
+const path = require('path');
+const bodyParser = require('body-parser');
+const fetch = require('node-fetch');
+const ejs = require('ejs');
+const pretty = require('express-prettify');
+
+const logger = log4js.getLogger(appName);
+const app = express();
+const server = http.createServer(app);
+
+const env = JSON.parse(process.env.VCAP_SERVICES);
+const credentials = env['pm-20'][0].credentials;
+const auth = 'Basic ' + new Buffer(credentials.username + ':' + credentials.password).toString('base64');
+
+var token = 'Bearer ';
+
+fetch('https://ibm-watson-ml.mybluemix.net/v3/identity/token', {
+  credentials: 'include',
+  headers: {
+    'Content-Type': 'application/json',
+    'Authorization': auth
+  }
+})
+.then(response => response.json())
+.then(json => token += json.token);
+
+// set the view engine to ejs
+app.set('view engine', 'ejs');
+
+// index page 
+app.get('/', function(req, res) {
+  res.render('pages/index');
+});
+
+app.use(log4js.connectLogger(logger, { level: process.env.LOG_LEVEL || 'info' }));
+const serviceManager = require('./services/service-manager');
+require('./services/index')(app);
+require('./routers/index')(app,server);
+
+// Add your code here
+
+const port = process.env.PORT || localConfig.port;
+server.listen(port, function(){
+  logger.info(`CustomerChurnWebapp listening on http://localhost:${port}/appmetrics-dash`);
+  
+  logger.info(`CustomerChurnWebapp listening on http://localhost:${port}`);
+});
+
+app.use(bodyParser.urlencoded({
+  extended: true
+}));
+app.use(bodyParser.json());
+
+app.post('/results', function(req, res) {
+  const tenure = req.body.tenure;
+  const security = req.body.security;
+  const support = req.body.support;
+  const contract = req.body.contract;
+  const dependents = req.body.dependents;
+  const phone = req.body.phone;
+  const internet = req.body.internet;
+  const payment = req.body.payment;
+  const backup = req.body.backup;
+  const charges = req.body.charges;
+  const protection = req.body.protection;
+  const paperless = req.body.paperless;
+
+  fetch('https://ibm-watson-ml.mybluemix.net/v3/wml_instances/461ebac2-ce8e-44e6-b3b9-b6f42a5f31b6/deployments/46608e13-988e-4e1b-969c-fe7bf8cf12c4/online', {
```

Comment:
This looks like a hard-code URL that is specific to your deployment.
I *think* it should use an environment variable, or other variable that points to a config file, so that it could work for anyone.